frontend/src/api/TokenAPI.ts

```diff
@@ -0,0 +1,30 @@
+const TOKEN_KEY = "token";
+
+export function getToken() {
+    let tokenResult = localStorage.getItem(TOKEN_KEY);
+    if (!tokenResult) return null;
+    try {
+        return JSON.parse(tokenResult);
+    } catch (error) {
+        console.log(error);
+        return null;
+    }
+}
+
+export function getTokenString() {
+    let tokenString = localStorage.getItem(TOKEN_KEY);
+    if (!tokenString) return null;
+    return tokenString;
+}
+
+export function setToken(token: {token: string, expiration: string}) {
```

Comment:
`expiration` is unused, is the intention to fill in that feature at a later date?